RiotSwiftUI/Modules/Template/SimpleUserProfileExample/Service/TemplateUserProfileServiceProtocol.swift

```diff
@@ -0,0 +1,36 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import Combine
+
+@available(iOS 14.0, *)
+protocol TemplateUserProfileServiceProtocol: Avatarable {
+    var userId: String { get }
+    var displayName: String? { get }
+    var avatarUrl: String? { get }
+    var presencePublisher: AnyPublisher<TemplateUserProfilePresence, Never> { get }
+}
+
+@available(iOS 14.0, *)
+extension TemplateUserProfileServiceProtocol {
```

Comment:
I can see why these are here, but it feels a bit weird.

Reply:
Will have a think